test/Connection.js

```diff
@@ -80,13 +88,98 @@ describe('Connection', () => {
     })
   }
 
+  const handlesErrors = (method, requestPath, sendRequest) => {
```

Comment:
@hiroara This test suite is somewhat redundant now and it can be reduced to just two test cases, one for errors that adhere to api error schema and other to the ones coming from AWS API Gateway. What do you think?

Reply:
I think that the number of current cases is fine :)
This library is small yet, and even in the future, I expect that it will be smaller than usual web apps. So the execution time of testing will not become huge.
In conclusion, we should increase test cases for now from my understanding.

Reply:
Great! I agree, execution time won't be a concern for a long time and more tests won't hurt as long as they are meaningful and these ones are :+1: